superset/models/dashboard.py

```diff
@@ -160,6 +160,7 @@ class Dashboard(Model, AuditMixinNullable, ImportExportMixin):
     published = Column(Boolean, default=False)
     is_managed_externally = Column(Boolean, nullable=False, default=False)
     external_url = Column(Text, nullable=True)
+    static_image = Column(Text, nullable=True)
```

Comment:
Shouldn't this be a `BLOB` (or similar) if we're storing a binary object. Additionally could you talk to any performance concerns this could have, i.e., the payload for fetching a record will likely increase significantly.

I'm not sure if having a separate table to house these images makes more sense. 

cc @betodealmeida @michael-s-molina